miden-tx/tests/common/mod.rs

```diff
@@ -78,3 +87,61 @@ impl DataStore for MockDataStore {
         Ok(self.account.code().module().clone())
     }
 }
+
+// HELPER FUNCTIONS
+// ================================================================================================
+pub fn get_new_key_pair_with_advice_map() -> (KeyPair, ([Felt; 4], Vec<Felt>)) {
+    let keypair: KeyPair = KeyPair::new().unwrap();
+
+    let pk: Word = keypair.public_key().into();
+    let pk_sk_bytes = keypair.to_bytes();
+    let to_adv_map = pk_sk_bytes.iter().map(|a| Felt::new(*a as u64)).collect::<Vec<Felt>>();
+    let advice_map_tupel: ([Felt; 4], Vec<Felt>) = (pk, to_adv_map.into());
+
+    (keypair, advice_map_tupel)
+}
+
+#[allow(dead_code)]
+pub fn get_account_with_default_account_code(
+    account_id: AccountId,
+    public_key: PublicKey,
```

Comment:
If we implement the changes described for key pair creation then this can be `public_key: Word`.